dask_sql/physical/rel/custom/create_experiment.py

```diff
@@ -147,6 +156,19 @@ def convert(self, rel: "LogicalPlan", context: "dask_sql.Context") -> DataContai
         y = training_df[target_column]
 
         if model_class and experiment_class:
+            if type(training_df) == dd.core.DataFrame:
+                if model_class in cpu_classes:
+                    model_class = cpu_classes[model_class]
+                if experiment_class in cpu_classes:
+                    experiment_class = cpu_classes[experiment_class]
+            elif (
+                dask_cudf is not None and type(training_df) == dask_cudf.core.DataFrame
```

Comment:
To avoid a dask-cudf import in this file, we could do:

```suggestion
                "cudf" in str(training_df._partition_type)
```

The motive here being that in the past, cuDF has failed to import on CPU machines where it was installed, which would then cause errors in dask-sql when we attempt to import it (xref #473)